stages/org.osbuild.test.timeout

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+"""
+Test stage that is used for timeout tests
+"""
+
+import time
+import sys
+
+def main():
+
+    print('Test stage output.')
+    time.sleep(2)
```

Comment:
I get it now, but I think I would have made this sleep forever, to make the tests more obvious. It would mean you cannot test the case that the stage finishes before the timeout, but all our other tests verifies that case anyway.